samples/cpp/ufacedetect.cpp

```diff
@@ -20,13 +20,13 @@ static void help()
             "This classifier can recognize many kinds of rigid objects, once the appropriate classifier is trained.\n"
             "It's most known use is for faces.\n"
             "Usage:\n"
-            "./facedetect [--cascade=<cascade_path> this is the primary trained classifier such as frontal face]\n"
+            "./ufacedetect [--cascade=<cascade_path> this is the primary trained classifier such as frontal face]\n"
```

Comment:
For me it is still not clear what the difference is between facedetect and ufacedetect sample. Shall we merge both samples together?